### ISSUE TITLE
Automate publishing to play store when a new tag is created

### ISSUE THREAD

**Issue by niqdev** [niqdev/ipcam-view#169]
See https://github.com/niqdev/ipcam-view/pull/167#issuecomment-986044584

**Comment by hannesa2** [comment]
The publish of current state as apk makes no sense, because camera url is hard coded and run into a timeout

**Comment by niqdev** [comment]
Fair enough, just keeping track of it in case the demo is converted to an actual app
